invoices/update.go

```diff
@@ -0,0 +1,319 @@
+package invoices
```

Comment:
invoices/update.go needs unit tests

Reply:
It is covered by `invoiceregistry_test.go`. There is indeed an opportunity to now test this code in isolation, but this PR also didn't introduce this code, just moved it.

Reply:
Sorry, missed that this is a dependent PR. Discussed this offline too, but to restate I'm on the opinion that separate units should be tested separately if possible.